net/nebula/files/nebula.proto

```diff
@@ -5,6 +5,8 @@ readonly PKG_VERSION='dev-test'
 readonly PROG=/usr/sbin/nebula
 readonly packageName='nebula-proto'
 
+# https://github.com/openwrt/openwrt/blob/fbf1835f46addf25a752a16be40d8f5617be3112/package/network/utils/umbim/files/lib/netifd/proto/mbim.sh
```

Comment:
This comment is somehow unclear for me. Can you add slight todo meaning for this one?